src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String indentation = "    ";
+    public static String doubleIndentation = indentation + "  ";
+    public static String tripleIndentation = doubleIndentation + "  ";
+    public static String horizontalLine = indentation
+            + "<------------------------------------------------------------>\n";
+    public static Task[] Tasks = new Task[100];
+    public static int numberOfTasks = 0;
+
+    public static void printStatement(String statement){
+        System.out.println(horizontalLine + indentation + "Here is yuqiaoluolong's Duke: \n"
+                + statement + horizontalLine);
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greet = indentation + " Hello! I'm Duke\n"
+                + indentation + " What can I do for you?\n";
+        String bye = indentation + " Bye. Hope to see you again soon!\n";
+        String wrongWarning = "Sorry, I don't know what's your command. Can you type again?\n";
```

Comment:
Same thing here - since the wrong warning is a constant string, should it be named WRONG_WARNING instead?